financial-services-accelerator/accelerators/fs-is/carbon-home/repository/conf/financial-services.xml

```diff
@@ -82,4 +82,31 @@
         <MaxConnections>2000</MaxConnections>
         <MaxConnectionsPerRoute>1500</MaxConnectionsPerRoute>
     </HTTPConnectionPool>
+    <EventNotifications>
+        <NotificationGeneration>
+            <NotificationGenerator>org.wso2.financial.services.accelerator.event.notifications.service.DefaultEventNotificationGenerator</NotificationGenerator>
+            <TokenIssuer>www.wso2.com</TokenIssuer>
+            <NumberOfSetsToReturn>5</NumberOfSetsToReturn>
+        </NotificationGeneration>
+        <EventCreationHandler>org.wso2.financial.services.accelerator.event.notifications.service.handler.DefaultEventCreationServiceHandler</EventCreationHandler>
+        <EventPollingHandler>org.wso2.financial.services.accelerator.event.notifications.service.handler.DefaultEventPollingServiceHandler</EventPollingHandler>
+        <EventSubscriptionHandler>org.wso2.financial.services.accelerator.event.notifications.service.handler.DefaultEventSubscriptionServiceHandler</EventSubscriptionHandler>
+        <PollingResponseParams>
+            <IsSubClaimAvailable>true</IsSubClaimAvailable>
+            <IsTxnClaimAvailable>{{financial_services.event.notifications.set_txn_claim_included}}</IsTxnClaimAvailable>
```

Comment:
Is this a mistake?

Reply:
Fixed in https://github.com/wso2/financial-services-accelerator/pull/159/commits/247fe15ce5007a468941c9ad2df3af6a7185c0fe